### PR TITLE
Heroku/feature

### PR DESCRIPTION
**Pull request by mitsuru-han**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1,2 @@
-release: bash ./release-tasks.sh
\ No newline at end of file
+release: bash ./release-tasks.sh
```
Comment (mitsuru-han) [inline Procfile:1]: uni
```suggestion
release: bash ./release-tasks.shAA
```

**Comment by handaman-job** [comment]
release: bash ./release-tasks.sh

**Comment by handaman-job** [comment]
release: bash ./release-tasks.sh

**Comment by mitsuru-han** [comment]
これはあれだねー
https://github.com/handaman-job/kadai-tasklist/blob/155f39ee50e0edb2dcbc7ee025f85cc0d0006627/Procfile#L1-L2

File: Procfile (hunk 1 above)
Comment (mitsuru-han) [inline Procfile:1]: うなうな

**Pull request merged**